taxi/views.py

```diff
@@ -85,6 +124,23 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super().get_context_data(**kwargs)
+        username = self.request.GET.get("username", "")
+        context["search_form"] = DriverSearchForm(
+            initial={"username": username}
+        )
+        return context
+
+    def get_queryset(self):
+        self.queryset = Driver.objects.prefetch_related("cars")
```

Comment:
It isn't recommended to do it (self.queryset = ...). get_queryset() should return queryset, but don't set it

You see, when django needs queryset, it calls get_queryset() method because you may need dynamic queryset depending on some conditions. Tis method should return queryset. In case this method is not overrided, django'll take queryset from queryset attribute